base/applications/network/ipconfig/lang/ro-RO.rc

```diff
@@ -55,38 +56,41 @@ END
 
 STRINGTABLE
 BEGIN
-    IDS_UNKNOWNADAPTER "\nUnknown Adapter: %ls\n"
-    IDS_OTHER "\nalt tip de adaptor: %ls\n"
-    IDS_ETH "\nadaptor Ethernet: %ls\n"
-    IDS_TOKEN "\nadaptor Token Ring: %ls\n"
-    IDS_FDDI "\nadaptor FDDI: %ls\n"
-    IDS_PPP "\nadaptor PPP: %ls\n"
-    IDS_LOOP "\nadaptor Loopback: %ls\n"
-    IDS_SLIP "\nadaptor SLIP: %ls\n"
-    IDS_WIFI "\nWireless Network Adapter: %ls\n"
+
+    IDS_UNKNOWNADAPTER "Adaptor necunoscut: %ls\n"
+    IDS_OTHER "\nAlt tip de adaptor: %ls\n"
+    IDS_ETH "\nAdaptor Ethernet: %ls\n"
+    IDS_TOKEN "\nAdaptor Token Ring: %ls\n"
+    IDS_FDDI "\nAdaptor FDDI: %ls\n"
+    IDS_PPP "\nAdaptor PPP: %ls\n"
+    IDS_LOOP "\nAdaptor Loopback: %ls\n"
+    IDS_SLIP "\nAdaptor SLIP: %ls\n"
+    IDS_WIFI "\nAdaptor de rețea fără fir: %ls\n"
+
+
 END
 
 STRINGTABLE
 BEGIN
     IDS_DNSNAME              "\t%ls\n"
     IDS_DNSLINE              "\t----------------------------------------\n"
-    IDS_DNSRECORDNAME        "\tRecord Name . . . . . : %ls\n"
-    IDS_DNSRECORDTYPE        "\tRecord Type . . . . . : %hu\n"
-    IDS_DNSRECORDTTL         "\tTime To Live. . . . . : %lu\n"
-    IDS_DNSRECORDLENGTH      "\tData Length . . . . . : %hu\n"
-    IDS_DNSSECTIONQUESTION   "\tSection . . . . . . . : Question\n"
-    IDS_DNSSECTIONANSWER     "\tSection . . . . . . . : Answer\n"
-    IDS_DNSSECTIONAUTHORITY  "\tSection . . . . . . . : Authority\n"
-    IDS_DNSSECTIONADDITIONAL "\tSection . . . . . . . : Additional\n"
-    IDS_DNSTYPEA             "\tA (Host) Record . . . : %ls\n"
-    IDS_DNSTYPENS            "\tNS Record . . . . . . : %ls\n"
-    IDS_DNSTYPECNAME         "\tCNAME Record. . . . . : %ls\n"
-    IDS_DNSTYPESOA           "\tSOA Record. . . . . . : \n"
-    IDS_DNSTYPEPTR           "\tPTR Record. . . . . . : %ls\n"
-    IDS_DNSTYPEMX            "\tMX Record . . . . . . : \n"
-    IDS_DNSTYPEAAAA          "\tAAAA Record . . . . . : %ls\n"
-    IDS_DNSTYPEATMA          "\tATMA Record . . . . . : \n"
-    IDS_DNSTYPESRV           "\tSRV Record. . . . . . : \n"
+    IDS_DNSRECORDNAME        "\tNume de înregistrare. . : %ls\n"
+    IDS_DNSRECORDTYPE        "\tTipul de înregistrare. .: %hu\n"
+    IDS_DNSRECORDTTL         "\tTimpul de trăit (TTL). .: %lu\n"
```

Comment:
Although I haven't comment before on this (and it still doesn't seem that important), I think it can be improved (like "timp de viață" or "valabilitate", maybe).